packages/template-typescript-minimal/app/ssr.js

```diff
@@ -40,7 +40,7 @@ const {handler} = runtime.createHandler(options, (app) => {
         res.send()
     })
 
-    app.get('/robots.txt', runtime.serveStaticFile('static/robots.txt'))
+    app.get('/favicon.ico', runtime.serveStaticFile('static/favicon.ico'))
```

Comment:
The remote server already has code to handle the `favicon`. So technically there are 2 handlers being setup for the when running remote, once here on line 43 of ssr.js and the second time inside the `build-remote-server.js` file below:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/9b255e7f04bfef4e6d398e2aa684677582ebb760/packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js#L572

We should probably do something about that.

Reply:
Also.. why are we removing the serving of the `robots.txt` file?

Reply:
Another reason why it might not make sense to have this line of code in ssr.js is because `faviconPath` is an option for the dev/remote servers. It would be odd to have that option but not do anything with it behind the scenes

Reply:
Good question... sorry i didn't explain that in my PR description.

Serving favicon is exactly the same as serving any static files. Since we are adding programmatic APIs like `serveStaticFile` and `serveServiceWorker`, I feel like it's better to keep things consistent, . If developers can already serve all static files including favicon via `serveStaticFile`, why do we need to have a specific configuration option `faviconPath`? It almost feels like the  `faviconPath` adds unnecessary complexity to the users.

So, I want to propose that we remove `faviconPath`, and since @shethj  is already working on the favicon ticket. I can work with @shethj to remove the option in his PR.

Reply:
> Also.. why are we removing the serving of the robots.txt file?

There is no robots.txt in the typescript template, and I feel it's unnecessary for a "minimal" template.

Reply:
I think it makes sense to have robots and favicon work the same way. So that is cool with me. You'd just have to make sure you remove all that special logic in the remote server factory. 

But one thing that I'm curious about is your implementation of the `serveStaticFile` for the dev server factory. It's completely different from the implementation in the remote server factory and I wasn't expecting that, since I assumed that the only real issue was the fact that the build dir was set to `build` for remote and it should be something like `` for development. Lets have a small chat, later and se can talk about that a bit.